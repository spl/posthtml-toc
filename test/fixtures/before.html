<html>
<body>
<h1>Title 1</h1>
<p>Text</p>
<h2 id="title2">Title 2</h2>
<p>Text</p>
</body>
</html>
