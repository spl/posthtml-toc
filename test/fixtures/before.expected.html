<html>
<body>
<h1>Title 1</h1>
<p>Text</p><div id="toc"><h2>Content</h2><ul><li><a href="#title2">Title 2</a></li></ul></div>
<h2 id="title2">Title 2</h2>
<p>Text</p>
</body>
</html>
